Sources/NIO/Embedded.swift

```diff
@@ -396,4 +396,18 @@ public class EmbeddedChannel: Channel {
         }
         fatalError("option \(option) not supported")
     }
+
+    public func bind(to address: SocketAddress, promise: EventLoopPromise<Void>?) {
+        promise?.futureResult.whenSuccess { _ in
```

Comment:
Nit: you shouldn't need a `_ in` here, as the return value of the `promise` is `Void`, which you're allowed to implicitly discard.

Reply:
Updated...